python/metatensor-operations/tests/detach.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import metatensor
+
+
+try:
+    import torch  # noqa
+
+    HAS_TORCH = True
+except ImportError:
+    HAS_TORCH = False
+
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "data")
+
+
+def test_detach():
+    tensor = metatensor.load(
+        os.path.join(DATA_ROOT, "qm7-power-spectrum.npz"),
+        # the npz is using DEFLATE compression, metatensor only supports STORED
+        use_numpy=True,
+    )
+
+    # just checking that everything runs fine with numpy
+    tensor = metatensor.detach(tensor)
+
+    if HAS_TORCH:
+        tensor = metatensor.to(tensor, backend="torch")
+        for block in tensor:
+            block.values.requires_grad_(True)
```

Comment:
Could this be made into its own convenience operation, to provide opposite functionality to `detach()`?

Reply:
done!